src/components/NotificationList.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { AlertList } from 'react-bs-notifier'
+import { uiNotificationDelete } from '../actions/ui'
+
+class NotificationList extends Component {
+
+  static propTypes = {
+    notifications: PropTypes.array
+  }
+
+  static defaultProps = {
+    notifications: []
+  }
+
+  render() {
+    const {dispatch} = this.props
+    return (
+      <div>
+        <AlertList
+          position="top-right"
+          alerts={this.props.notifications}
+          timeout={4000}
```

Comment:
make this a prop? with a default?

Reply:
Sure for consistency. But if you wanted to change this for a specific alert you wouldn't use a prop, you pass it in the object to the action:
```js
dispatch(uiNotificationNew({ type: 'info', message: 'foo', timeout: 15000 }))
```

Reply:
Actually, I should remove it the default! The timeout specifies when it should auto-hide. With the default in place you can never create a notification that doesn't auto-hide (eg. a failure you want the user to acknowledge).